pkg/remote/aws/repository/elb_repository.go

```diff
@@ -0,0 +1,46 @@
+package repository
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/elb/elbiface"
+	"github.com/snyk/driftctl/pkg/remote/cache"
+)
+
+type ELBRepository interface {
+	ListAllLoadBalancers() ([]*elb.LoadBalancerDescription, error)
+}
+
+type elbRepository struct {
+	client elbiface.ELBAPI
+	cache  cache.Cache
+}
+
+func NewELBRepository(session *session.Session, c cache.Cache) *elbRepository {
+	return &elbRepository{
+		elb.New(session),
+		c,
+	}
+}
+
+func (r *elbRepository) ListAllLoadBalancers() ([]*elb.LoadBalancerDescription, error) {
+	cacheKey := "elbListAllLoadBalancers"
+	v := r.cache.GetAndLock(cacheKey)
+	defer r.cache.Unlock(cacheKey)
```

Comment:
It's not really mandatory right now to do this (GetAndLock and defer the Unlock) since we're not using this method in any other enumerator.